### PR TITLE
Jenkins sync and fixes

### PR DESCRIPTION
**Pull request by palvarez89**

### CODE REVIEW

File: roles/jenkins/files/jobs/Echaloasuerte-PROD-Deploy/config.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (Mariocj89) [inline roles/jenkins/files/jobs/Echaloasuerte-PROD-Deploy/config.xml]: this job is not working :)

Comment (palvarez89) [inline roles/jenkins/files/jobs/Echaloasuerte-PROD-Deploy/config.xml]: Yes, I was aware of that, and I also thought looking at the failure that the problem was easy to solve.

Now I think that the main problem is that our devel server can't access to our production server if we don't provide it with some ssh keys, but I think this is not a good practice given that we double de chance of getting the production server compromised.

Meh...

Comment (Mariocj89) [inline roles/jenkins/files/jobs/Echaloasuerte-PROD-Deploy/config.xml]: should we remove it then? :)x

**Comment by palvarez89** [comment]
removed!

**Pull request merged**